src/main/java/nl/hsac/fitnesse/fixture/slim/web/BrowserTest.java

```diff
@@ -1374,7 +1379,7 @@ private String getScreenshotLink(String screenshotFile) {
         return screenshotFile;
     }
 
-    private String createScreenshot(String basename) {
+    public String createScreenshot(String basename) {
```

Comment:
Making this protected would suffice, right? This is not something to be called from the wiki.